src/main/java/com/alkemy/ong/controller/v1/NewController.java

```diff
@@ -33,4 +37,10 @@
 		return new ResponseEntity<NewDetailDto>(service.addNews(news,id),HttpStatus.CREATED);
 	}
 
+    @PostMapping
+    @ResponseStatus(value = HttpStatus.CREATED)
```

Comment:
Please, return the id of the New in the Location Header. Use the code bellow:
![imagen](https://user-images.githubusercontent.com/21185543/157106612-bf3d84c0-c384-452f-ac1f-5de395064112.png)